--- a/tools/gen_cc_proto_descriptor.py
+++ b/tools/gen_cc_proto_descriptor.py
@@ -16,6 +16,7 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
+import re
 import os
 import sys
 import argparse
@@ -38,8 +39,11 @@
       binary, width=80, initial_indent='    ', subsequent_indent='     ')
 
   relative_target = os.path.relpath(target, gendir)
-  include_guard = relative_target.replace('\\', '_').replace('/', '_').replace(
-      '.', '_').upper() + '_'
+  include_guard = re.sub(
+      r'[^A-Z0-9_]',
+      '_',
+      relative_target.replace('\\', '_').replace('/', '_').replace('.', '_').upper(),
+  ) + '_'
 
   with open(target, 'wb') as f:
     f.write("""/*
